runtime/near-vm-runner/tests/test-contract-rs/src/lib.rs

```diff
@@ -370,12 +333,13 @@ pub unsafe fn sum_n() {
     read_register(0, data.as_ptr() as u64);
     let n = u64::from_le_bytes(data);
 
-    let mut sum = 0u64;
-    for i in 0..n {
-        sum += i;
+    let mut sum = 0u128;
+    for i in 1..n + 1 {
+        sum += (i * i) as u128;
     }
+    sum /= n as u128;
```

Comment:
Why this line needed?

Reply:
Otherwise it overfits `u64` for larger sums